mix.exs

```diff
@@ -36,7 +36,7 @@ defmodule Lob.Mixfile do
       {:credo, "~> 0.8", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 0.5.1", only: [:dev, :test], runtime: false},
       {:excoveralls, "~> 0.7.4", only: :test},
-      {:httpoison, "~> 0.13"},
+      {:httpoison, "~> 1.0"},
```

Comment:
Is there a minimum 1.x version you want to support? I am choosing anything in the 1.x series right now.

Reply:
`1.0` is good for now, looks like `1.2.0` bumps the minimum Elixir version to `1.5` which conflicts with this package's min version.

Reply:
Great. Is there anything else you would like me to do?